### PR TITLE
Add Rest Multipart Form Post and Put examples

### PR DESCRIPTION
**Pull request by nebilbenmabrouk**

### CODE REVIEW

**Comment by nebilbenmabrouk** [comment]
> @nebilbenmabrouk in your PR the previous workflows were not removed

Yes they were not removed, they were renamed (see below):

`       modified:   Rest/METADATA.json
	new file:   Rest/resources/catalog/Multipart_Form_Post.xml
	new file:   Rest/resources/catalog/Multipart_Form_Put.xml
	renamed:    Rest/resources/catalog/Form_Post.xml -> Rest/resources/catalog/UrlEncoded_Form_Post.xml
	renamed:    Rest/resources/catalog/Form_Put.xml -> Rest/resources/catalog/UrlEncoded_Form_Put.xml
`

**Comment by fviale** [comment]
@nebilbenmabrouk ah ok, I see it now

File: Rest/resources/catalog/UrlEncoded_Form_Put.xml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+  xmlns="urn:proactive:jobdescriptor:3.14" xsi:schemaLocation="urn:proactive:jobdescriptor:3.14 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.14/schedulerjob.xsd"  name="UrlEncoded_Form_Put" projectName="Form Parameters" tags="REST,API,Web Service" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2"  >
   <variables>
     <variable name="ENDPOINT" value="https://postman-echo.com" model="PA:URL" description="Base URL of the server or service that will be queried." group="Http Connection" advanced="false" hidden="false"/>
-    <variable name="USER" value="" model="" description="If the service requires basic http authentication, enter a username." group="Http Connection" advanced="true" hidden="false"/>
+    <variable name="USER" value=""  description="If the service requires basic http authentication, enter a username." group="Http Connection" advanced="true" hidden="false"/>
```
Comment (fviale) [inline Rest/resources/catalog/UrlEncoded_Form_Put.xml:7]: same comment as above regarding this workflow description

File: Rest/resources/catalog/Body_Patch.xml [hunk 2]
```diff
@@ -8,21 +8,21 @@
...
+    <![CDATA[ A workflow that executes a REST request using the method PATCH in order to partially update some data. ] ]]>
   </description>
   <genericInformation>
     <info name="bucketName" value="communication-rest"/>
     <info name="workflow.icon" value="/automation-dashboard/styles/patterns/img/wf-icons/api-rest.png"/>
     <info name="group" value="public-objects"/>
   </genericInformation>
   <taskFlow>
-    <task name="RestBodyPatch"
+    <task name="Rest-Body-Patch"
```
Comment (fviale) [inline Rest/resources/catalog/Body_Patch.xml:19]: could you please use underscore instead of dash in new task names ?

Comment (fviale) [reply Rest/resources/catalog/Body_Patch.xml:19]: not really, we never use dash in task names. We don't have the convention that you describe, so please don't invent one today ;-)

File: Rest/resources/catalog/UrlEncoded_Form_Put.xml [hunk 3]
```diff
@@ -230,27 +210,55 @@ switch (variables.get("RESPONSE_FORMAT")) {
...
-      xmlns="http://www.w3.org/1999/xhtml"><circle cx="10" cy="10" r="10" version="1.1"
-      xmlns="http://www.w3.org/1999/xhtml" fill="#666" stroke="none" style=""></circle></svg></div></div></div></body></html>
+        </style>
+    </head>
+    <body>
+    <div id="workflow-visualization-view"><div id="workflow-visualization" style="position:relative;top:-291.453125px;left:-788.90625px"><div class="task _jsPlumb_endpoint_anchor_ ui-draggable" id="jsPlumb_1_88" style="top: 296.453px; left: 793.906px;"><a class="task-name" data-toggle="tooltip" data-placement="right" title="A task performing a rest PUT request using form data.
+This template supports only basic authentication, for more advanced authentication settings, please modify the template according to the rest-assured documentation:
+https://github.com/rest-assured/rest-assured/wiki/Usage#authentication
+
+It accepts the following parameters:
```
Comment (fviale) [inline Rest/resources/catalog/UrlEncoded_Form_Put.xml:240]: it's strange that the visualization contains the old description that you just modified. Could you try to reload the workflow and then export it again to see if this disappears ?

File: Rest/resources/catalog/Multipart_Form_Put.xml [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+            width:2736px;
+            height:3132px;
+            }
+        </style>
+    </head>
+    <body>
+    <div id="workflow-visualization-view"><div id="workflow-visualization" style="position:relative;top:-367.484375px;left:-583.484375px"><div class="task _jsPlumb_endpoint_anchor_ ui-draggable" id="jsPlumb_1_121" style="top: 372.484px; left: 588.484px;"><a class="task-name" data-toggle="tooltip" data-placement="right" title="A task performing a rest PUT request using form data.
+This template supports only basic authentication, for more advanced authentication settings, please modify the template according to the rest-assured documentation:
+https://github.com/rest-assured/rest-assured/wiki/Usage#authentication
+
```
Comment (fviale) [inline Rest/resources/catalog/Multipart_Form_Put.xml:239]: the description in the metada section here is strange

File: Rest/resources/catalog/Body_Patch.xml (hunk 2 above)
Comment (nebilbenmabrouk) [reply Rest/resources/catalog/Body_Patch.xml]: OK, done.

File: Rest/resources/catalog/UrlEncoded_Form_Put.xml (hunk 3 above)
Comment (nebilbenmabrouk) [reply Rest/resources/catalog/UrlEncoded_Form_Put.xml]: Loaded in the studio and now it is up-to-date.

File: Rest/resources/catalog/Multipart_Form_Put.xml (hunk 4 above)
Comment (nebilbenmabrouk) [reply Rest/resources/catalog/Multipart_Form_Put.xml]: Loaded in the studio and now it is up-to-date.

**Comment by fviale** [comment]
Retest this please

**Pull request merged**